src/Illuminate/Queue/RedisQueue.php

```diff
@@ -47,72 +47,72 @@ class RedisQueue extends Queue implements QueueContract
     /**
      * Create a new Redis queue instance.
      *
-     * @param  \Illuminate\Contracts\Redis\Factory  $redis
-     * @param  string  $default
-     * @param  string  $connection
-     * @param  int  $retryAfter
-     * @param  int|null  $blockFor
+     * @param  \Illuminate\Contracts\Redis\Factory $redis
+     * @param  string                              $default
```

Comment:
Please keep 2 spaces in thr phpdoc for param annotations, like it was before, everywhere.